test/gtest/CMakeLists.txt

```diff
@@ -15,6 +15,14 @@ if(MIOPEN_BACKEND_OPENCL)
   set(SKIP_TESTS dumpTensorTest)
 endif()
 
+set(SOURCES ${SOURCES}
+    "${CMAKE_CURRENT_SOURCE_DIR}/log_test_helper.cpp"
+    )
+
+if(MIOPEN_BACKEND_OPENCL)
+  set(SKIP_TESTS dumpTensorTest)
+endif()
+
```

Comment:
This is not related to this PR.

Reply:
Thanks for the comments. This is related to newly added tests since this branch is a bit dated.

Reply:
I do not understand how dumpTensorTest is related to cache test. Please clearly indicate this change in the PR title and description or move it into separate PR. Thanks!

Reply:
Understood your concern. dumpTensorTest has been merged and this branch was dated. Trying to merge this branch back to develop and thus dumpTensorTest is pulled locally.

Reply:
@xinlipn Please help us understand why this needed to be added to this PR?

Reply:
![image](https://user-images.githubusercontent.com/12038446/197241357-501017ad-787b-4e64-9d59-7b38eb9134c2.png)

@JehandadKhan will remove duplication. But if I delete dumpTensorTest and log_test_helper.cp related changes all together. There will be make errors since this branch is dated and new files had been merged to this branch